modules/sdk-core/src/bitgo/inscription/inscription.ts

```diff
@@ -0,0 +1,76 @@
+import assert from 'assert';
+import { IInscription, SubmitTransactionResponse } from './iInscription';
+import { BitGoBase } from '../bitgoBase';
+import { IWallet } from '../wallet';
+import { inscriptions, PreparedInscriptionRevealData, TapLeafScript } from '@bitgo/utxo-ord';
+import { KeyIndices } from '../keychain';
+import { xprvToRawPrv } from '../../account-lib';
+import { HalfSignedUtxoTransaction } from '../baseCoin';
+import { inferNetworkFromCoinName } from '../utils';
+import * as utxolib from '@bitgo/utxo-lib';
+
+export class Inscription implements IInscription {
```

Comment:
```suggestion
export class InscriptionBuilder implements IInscriptionBuilder {
```

IF we actually need the interface - still unclear why

I don't understand why we just add a `class InscriptionBuilder` that lives in `modules/abstract-utxo/`